docs/pages/reference/signature-algorithms.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Signature Algorithms
+h1: Signature Algorithms Reference
+description: "Signature algorithms used in Teleport."
+---
+
+An important role of the Teleport Auth Service is issuing short-lived SSH and
+TLS certificates to users and hosts that allow all connections to be
+authenticated, authorized, and encrypted.
+This page describes the various cryptographic signature algorithms used to sign
+each kind of certificate issued by Teleport.
+
+Historically, every Teleport-issued certificate was signed by a 2048-bit RSA key.
+RSA is one of the oldest and most well-known asymmetric cryptography algorithms.
+While it is still considered secure (as long as your keys are large enough),
+modern alternatives based on Elliptic-Curve Cryptography offer much better
+performance with smaller keys and better security.
+
+## Signature algorithm suites
+
+Unilaterally changing the signature algorithms used in Teleport would be a
+breaking change for our users, as certain environments may expect or rely on RSA
+signatures.
+Some users may also require different algorithms to be used if they are looking
+for FIPS compliance or they are storing key material in a Hardware Security
+Module (HSM) or Key Management Service (KMS) that only supports certain
+algorithms.
+
+To allow new and existing Teleport users to opt-in to the new signature
+algorithms when they are ready, we introduced the concept of "signature
+algorithm suites".
```

Comment:
```suggestion
To allow new and existing Teleport users to opt in to the new signature
algorithms when they are ready, we introduced the concept of **signature
algorithm suites**.
```
- Removing unnecessary hyphen
- Per the style guide (github.com/gravitational/docs), we use bold instead of quotes for introducing key terms